server/package.json

```diff
@@ -12,8 +12,7 @@
     "create-migration": "knex migrate:make $1",
     "create-seed": "knex seed:make $1",
     "rollback": "knex migrate:rollback",
-    "recreate-db": "npm run rollback && npm run migrate",
-    "circle-ci-recreate-db": "npm run rollback && npm run migrate"
+    "recreate-db": "npm run rollback && npm run migrate"
```

Comment:
I thought having a different one for CI, like `ci-create-db` - it will be the same now, but so that we just change in package.json without having to change configuration in heroku

Reply:
scratch that .. we have to change ProcFile anyhow, it is not a configuration in heroku